app/lib/submission_builder/ty2024/states/nj/documents/nj1040.rb

```diff
@@ -50,9 +50,15 @@ def document
                       raise "Filing status not found"
                     end
                   end
-                  
+
                   xml.Exemptions do
                     xml.YouRegular "X"
+                    xml.SpouseCURegular calculated_fields.fetch(:NJ1040_LINE_6_SPOUSE) ? "X" : ""
+                    xml.DomesticPartnerRegular ""
+                    xml.Line6 calculated_fields.fetch(:NJ1040_LINE_6)
```

Comment:
[boulder] Is Line6 a valid XML element per our schemas? Same question for Line7

Reply:
Wow, thanks for catching that. I misread the schema - it is not. It appears that the XML simply has fields for each checkbox. With this in mind, my plan is still to use the calculator to generate calculated values for lines 6-12, store them in `calculated_fields` and then sum them for Line 13 `TotalExemptionAmountA`, which is a valid field.